plugins/com.google.cloud.tools.eclipse.util/src/com/google/cloud/tools/eclipse/util/JavaPackageValidator.java

```diff
@@ -23,24 +23,21 @@
 
 public class JavaPackageValidator {
 
-  private static final String PLUGIN_ID = 
-      "com.google.cloud.tools.eclipse.appengine.newproject.AppEngineStandard"; //$NON-NLS-1$
-  
   /**
    * Check if a string is a legal Java package name.
    */
-  public static IStatus validate(String packageName) {
+  public static IStatus validate(String packageName, String pluginId) {
     if (packageName == null) {
-      return new Status(IStatus.ERROR, PLUGIN_ID, 45, "null package name", null);
+      return new Status(IStatus.ERROR, pluginId, 45, "null package name", null);
```

Comment:
I think it's totally reasonable to put a PLUGIN_ID of the `.util` bundle instead of getting it from callers. The error status will still clearly indicate that the error is from one of our CT4E bundles. It seems unnecessary to force callers to pass the string.